### DISCUSSION TITLE
arm64遇到了运行问题

### DISCUSSION THREAD

**Discussion by Experienceball** [natfrp/wiki#177] [使用问题] [answered]
在aidlux运行arm64版本的frp出现Segmentation fault错误，无法运行

**Comment by fakeboboliu** [comment, marked as answer]
请尝试下载时在 URL 后面加 `_noupx` 下载无 upx 版本。
另：我个人不推荐使用 AidLux，请允许我推荐 [Termux](https://github.com/termux/termux-app)
